src/wasm/cpp/build.sh

```diff
@@ -1,11 +1,8 @@
 #!/bin/bash
 
-set -e
-
 invoke() {
   # export EMCC_DEBUG=1 for debug info
-  em++ -s "EXPORTED_FUNCTIONS=['_malloc', '_free']" -s EMIT_EMSCRIPTEN_METADATA \
-       -std=c++17 -O3 test-particle.cc -o $1
+  em++ -s EMIT_EMSCRIPTEN_METADATA -std=c++17 -O3 test-particle.cc -o $1
```

Comment:
@mykmartin should review this.

Reply:
This is the result of a bad merge, I need to rebase the C++ off of head.

All that should have happened was the move of all C++ files to src/wasm/cpp with no internal changes.